packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -146,35 +167,88 @@ export class GCFBootstrapper {
     return config as Options;
   }
 
+  parseRequestHeaders(
+    request: express.Request
+  ): {name: string; id: string; signature: string; taskId: string} {
+    const name =
+      request.get('x-github-event') || request.get('X-GitHub-Event') || '';
+    const id =
+      request.get('x-github-delivery') ||
+      request.get('X-GitHub-Delivery') ||
+      '';
+    // TODO: add test to validate this signature is used on the initial
+    // webhook from GitHub:
+    const signature =
+      request.get('x-github-delivery') ||
+      request.get('X-GitHub-Delivery') ||
+      '';
+    const taskId =
+      request.get('X-CloudTasks-TaskName') ||
+      request.get('x-cloudtasks-taskname') ||
+      '';
+    return {name, id, signature, taskId};
+  }
+
+  parseTriggerType(name: string, taskId: string): TriggerType {
```

Comment:
Again, existing logic that I just pulled into a separate function and added `TriggerType`s

Reply:
This can probably be static, private, or live outside the class completely.